data/supporters.json

```diff
@@ -1,5 +1,18 @@
 {
   "usergroups": [
+    {
+      "name": "dotfmp.berlin",
+      "city": "Berlin",
+      "country": "Germany",
+      "link": "https://dotfmp.berlin",
+      "twitter": "dotfmp",
+      "contacts": [
+        {
+          "name": "pixi",
+          "email": "coc@dotfmp.berlin",
```

Comment:
Please check for valid JSON syntax (no `,`) :)

Reply:
Too quick c&p, sorry ;-)